index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Amr Ahmed - Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+            <div class="header">
+                <a href="#">Amr Ahmed</a>
+                <nav class="hamburger-menu">
+                    <i class="fas fa-bars">
+                    </i>
+                </nav>
+            </div>
```

Comment:
- [ ] It would be so much better to use a `header` tag instead of a `div`. This tag is meant to help with the structure of the HTML. Please use the header tag to wrap inside the logo and the hamburger button. 👍 🥇